src/lib/models/document.coffee

```diff
@@ -366,9 +466,18 @@ class DocumentModel extends FileModel
 		# Chain
 		@
 
-	# Get Eve
-	# Get the most ancestoral layout we have (the very top one)
+	###*
+	# Get the most ancestoral (root) layout we 
+	# have - ie, the very top one. Often this
+	# will be the base or default layout for
+	# a project. The layout where the head and other
+	# html on all pages is defined. In some project,
+	# however, there may be more than one root layout
+	# so we can't assume there will always only be one.
 	# next(err,layout)
+	# @method getEve
+	# @param {Object} next
+	###
```

Comment:
It could be mentioned that the use of this is to determine the output extension of the document.